integration-tests/smoke/ccip/ccip_rmn_test.go

```diff
@@ -561,6 +638,36 @@ func (tc rmnTestCase) killMarkedRmnNodes(t *testing.T, rmnCluster devenv.RMNClus
 	}
 }
 
+func (tc rmnTestCase) restartNode(t *testing.T, rmnCluster devenv.RMNCluster) func() {
```

Comment:
Btw, you can do it [simpler](https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/framework/examples/myproject/chaos/chaos_docker_test.go#L44) and test even more chaos cases with Docker if you need.